### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by ShaliniNayan**
Hello,

For this PR, I have a Bookstores initialize project using:

- Setup new React application using Create React App
- Created a new directory called Components.
- Install React Router V6.
- Set router inside of <App>
- Add 2 routes in your application, each should render a page component:
       -  /
       - /categories
- Created components inside of the /components directory:
       - Contains individual book states, as well as a button to delete the book
       - Contains a list to render individual books
       - Contains a form to create new books
       - Contains navigation to link to the 2 routes you created 
- Gitflow follows.
- No linter error.

### CODE REVIEW

File: src/components/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function HomePage() {
+  return (
+    <div>
+      <h1>Home Page</h1>
+    </div>
+  );
+}
```
Comment (andersonlebon) [inline src/components/HomePage.js]: - [x] Kindly make sure this routes display the component that contains all the books. hint 👉 
```js
function HomePage() {
  return (
    <Books />  // where the Books component renders the <BookList/> component and the <BookForm/> component
  );
}
```

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @ShaliniNayan 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## To highlight :champagne: 
- Your README file is now professional :heavy_check_mark: 
- Nice use of props and PropTypes ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [x] Kindly make sure the commit message are meaningfull, they should be talk about the change introduced in the commit and be unique.

image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/HomePage.js (hunk 1 above)
Comment (ShaliniNayan) [reply src/components/HomePage.js]: Thanks & Noted

**Review by julie-ify — APPROVED** [review]
Hi @ShaliniNayan,

## Highlights!
- Books component is index route
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**